build.sh

```diff
@@ -52,11 +52,11 @@ generate_rpm_sources() {
             echo "Generating $TARBALL from $SOURCE_TAG tag"
         fi
 
-        pushd "$SCR_DIR"
+        pushd "$SRC_DIR"
         git \
             archive \
             --format=tar.gz \
-            --prefix $NAME-$VERSION/ \
+            --prefix $NAME-$VERSION/jss/ \
```

Comment:
This is OK for 4.4 branch, but for 4.5 or later I'd rather not use the extra "jss" subdirectory because it's not really necessary, inconsistent with the other projects, and would require us to attach the tarball into GitHub release (so the tarball is not guaranteed to be correct).

Reply:
Also, without the "jss" subdirectory the patches can be generated easily with git format-patch (instead of diff) and applied with git am (instead of patch or git apply).

Reply:
So as far as I can tell.. There's two ways to fix this problem: downstream, or upstream. There'll be no more Fedora releases on this branch, so we only have to worry about aligning `v4.4.x` with `rhel-7.y` spec files. I'm operating under the (perhaps incorrect) assumption that the contents of the downstream spec files are there for a reason, perhaps something to do with tooling there (perhaps @mharmsen99 can weigh in).

Operating under that assumption... We only need the `jss/` dir on `v4.4.x` releases. On new `master` (`v4.5.x`) releases, in both upstream, Fedora, and newer RHEL releases, we don't need the `jss-<version>/jss/` prefix; only `jss-<version>`, and this is consistent everywhere.

*Only* on `v4.4.x` do we have to worry about the `jss/` subfolder requirement, and the additional upload to GitHub.